random_menu.rb

```diff
@@ -0,0 +1,57 @@
+foods = [
+  "arepas",
+  "chorizos",
+  "steak",
+  "tomato soup",
+  "eggs",
+  "bread",
+  "pasta",
+  "congee",
+  "pho",
+  "rolls"
+]
+
+styles = [
+  "deep fried",
+  "fried",
+  "grilled",
+  "boiled",
+  "pan fried",
+  "charbroiled",
+  "infusion",
+  "microwaved",
+  "marinated",
+  "wok cooked"
+]
+
+adjectives = [
+  "acidic",
+  "bitter",
+  "creamy",
+  "crispy",
+  "juicy",
+  "crumbly",
+  "salty",
+  "sour",
+  "ripe",
+  "yummy"
+]
+
+adjectives = adjectives.shuffle
```

Comment:
Nice use of shuffle! This is a clever way to get a random order without destroying the array as you go along.